src/Pods/Data/Map_Field_Values.php

```diff
@@ -703,6 +704,85 @@ public function image_fields( $field, $traverse, $field_data, $obj ) {
 		return pods_traverse( $traverse_params, $value );
 	}
 
+	/**
+	 * Map the date field value.
+	 *
+	 * @since 2.9.14
+	 *
+	 * @param string                  $field      The first field name in the path.
+	 * @param string[]                $traverse   The list of fields in the path excluding the first field name.
+	 * @param null|Field|Object_Field $field_data The field data or null if not a field.
+	 * @param Pods|null               $obj        The Pods object or null if not set.
+	 *
+	 * @return null|mixed The date field value or null if there was no match.
+	 */
+	public function date_fields( $field, $traverse, $field_data, $obj = null ) {
+
+		// Default date field handlers.
+		$date_fields = array(
+			'date',
+			'time',
+			'datetime',
+		);
+
+		// Skip if the field exists.
+		if ( $field_data && ! in_array( $field_data->get_type(), $date_fields, true ) ) {
+			return null;
+		}
+
+		$object_type = $obj ? $obj->pod_data->get_type() : null;
+
+		if ( 'post_type' === $object_type ) {
+			$date_fields[] = 'post_date';
+			$date_fields[] = 'post_date_gmt';
+			$date_fields[] = 'post_modified';
+			$date_fields[] = 'post_modified_gmt';
+		}
+
+		// Handle special field tags.
+		if ( ! in_array( $field, $date_fields, true ) ) {
+			return null;
+		}
+
+		$value = $obj->field( $field, array( 'raw' => true ) );
```

Comment:
We need a case for current date/time/datetime values.

```suggestion
		// Handle getting current field value.
		if ( $field_data ) {
			$value = $obj->field( $field, array( 'raw' => true ) );
		} else {
			$value = current_time( 'mysql' );
		}
```